stories/features/rewards/concepts.tsx

```diff
@@ -113,7 +118,7 @@ storiesOf('Feature Components/Rewards/Concepts/Desktop', module)
               <SiteBanner
                 domain={text('Domain', 'duckduckgo.com')}
                 name={text('Name', 'duckduckgo.com')}
-                screenName={screenName}
+                screenName={screenName || ''}
```

Comment:
is this needed? is it possible to be undefined?

Reply:
No. Removed